pymbolic/mapper/c_code.py

```diff
@@ -44,15 +44,15 @@ class CCodeMapper(SimplifyingSortingStringifyMapper):
     .. doctest::
 
         >>> import pymbolic.primitives as p
+        >>> CSE = p.make_common_subexpression
         >>> x = p.Variable("x")
-        >>> CSE = p.CommonSubexpression
         >>> u = CSE(3*x**2-5, "u")
         >>> expr = u/(u+3)*(u+5)
         >>> print(expr)
         (CSE(3*x**2 + -5) / (CSE(3*x**2 + -5) + 3))*(CSE(3*x**2 + -5) + 5)
 
     Notice that if we were to directly generate code from this, the
-    subexpression *u* would be evaluated multiple times.
+    subexpression *u* would not be evaluated multiple times.
```

Comment:
? Is this right?

Reply:
Ah, I see! Reworded a bit more verbosely in
https://github.com/inducer/pymbolic/compare/3bcf915c6b2a360eb09b98c7aba4d2641cfccf4e..451d51fe3936d7d157de7ab766d7a0d1bda65518